corehq/apps/email/migrations/0001_initial.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.20 on 2023-09-05 05:35
+
+from django.db import migrations, models
+import django.db.models.deletion
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SQLEmailSMTPBackend',
```

Comment:
I think references to `SQLEmailSMTPBackend` need to be updated to `EmailSMTPBackend`

Reply:
https://github.com/dimagi/commcare-hq/pull/33386/commits/e75ea28c303980bb0975f71d7e44356c40717767